.github/workflows/build.yml

```diff
@@ -80,10 +81,10 @@ jobs:
           prerelease: true
```

Comment:
there's an if on matrix.os on line 73 that will throw away binaries for 22.04 - why prepare them? - as test?